kyuubi-common/src/main/scala/org/apache/kyuubi/service/authentication/EngineSecureAuthenticationProviderImpl.scala

```diff
@@ -17,8 +17,12 @@
 
 package org.apache.kyuubi.service.authentication
 
-class EngineSecureAuthenticationProviderImpl extends PasswdAuthenticationProvider {
+class EngineSecureAuthenticationProviderImpl extends PasswdAuthenticationProvider
+  with TokenAuthenticationProvider {
   override def authenticate(user: String, password: String): Unit = {
     InternalSecurityAccessor.get().authToken(password)
   }
+  override def authenticate(token: String): Unit = {
+    InternalSecurityAccessor.get().authToken(token)
+  }
```

Comment:
not needed for `EngineSecureAuthenticationProviderImpl`.

EngineSecureAuthenticationProviderImpl only for `PasswdAuthenticationProvider`.